modules/ITK/cli/statismo-build-deformation-model.cxx

```diff
@@ -0,0 +1,142 @@
+#include <itkDirectory.h>
+#include <itkImageFileReader.h>
+
+#include "itkDataManager.h"
+#include "itkPCAModelBuilder.h"
+#include "itkStandardImageRepresenter.h"
+#include "itkStatisticalModel.h"
+
+#include <boost/program_options.hpp>
+
+#include "statismo-build-models-utils.h"
+
+namespace po = boost::program_options;
+
+
+struct programOptions {
+    bool bDisplayHelp;
+    string strDataListFile;
+    string strOutputFileName;
+    float fNoiseVariance;
+    bool bAutoNoise;
+};
+
+po::options_description initializeProgramOptions(programOptions& poParameters);
+bool isOptionsConflictPresent(programOptions& opt);
+void buildAndSaveDeformationModel(programOptions opt);
+
+
+
+int main(int argc, char** argv) {
+    programOptions poParameters;
+
+    po::positional_options_description optPositional;
+    optPositional.add("output-file", 1);
+    po::options_description optAllOptions = initializeProgramOptions(poParameters);
+
+
+    po::variables_map vm;
+    try {
+        po::parsed_options parsedOptions = po::command_line_parser(argc, argv).options(optAllOptions).positional(optPositional).run();
+        po::store(parsedOptions, vm);
+        po::notify(vm);
+    } catch (po::error& e) {
+        cerr << "An exception occurred while parsing the Command line:"<<endl;
```

Comment:
if this works, it means there is a global using namespace std, which is bad pratice... Where is it?